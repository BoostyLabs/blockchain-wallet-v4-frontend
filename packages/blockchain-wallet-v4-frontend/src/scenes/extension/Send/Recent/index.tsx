import React from 'react'
import { WalletsWrapper } from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send'
import WalletItem from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send/WalletItem'

const Recent: React.FC = () => {
  // TODO: mock recent wallets data.
  const recentWallets: string[] = [
    'J4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77',
    'J4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77'
  ]

  return (
    <WalletsWrapper>
      {recentWallets.map((address: string) => (
        <WalletItem walletAddress={address} key={address} />
      ))}
    </WalletsWrapper>
  )
}

export default Recent
