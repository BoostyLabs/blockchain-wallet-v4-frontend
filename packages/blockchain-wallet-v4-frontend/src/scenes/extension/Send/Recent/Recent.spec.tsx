import React from 'react'
import Recent from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send/Recent'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Recent extension tab', (): void => {
  const container = shallow(<Recent />)
  it('Recent tab renders correctly', (): void => {
    const tree = toJson(container)
    expect(tree).toMatchSnapshot()
  })
})
