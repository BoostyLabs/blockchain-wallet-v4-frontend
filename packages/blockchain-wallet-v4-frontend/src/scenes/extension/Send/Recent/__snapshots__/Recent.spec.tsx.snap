// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recent extension tab Recent tab renders correctly 1`] = `
<styled.div>
  <WalletItem
    key="J4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
    walletAddress="J4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
  />
  <WalletItem
    key="J4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
    walletAddress="J4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
  />
</styled.div>
`;
